packages/lib/modules/pool/actions/unstake/UnstakeProvider.tsx

```diff
@@ -35,14 +35,13 @@ export function _useUnstake() {
     isLoading: isLoadingClaims,
   } = useClaimsData([pool] as unknown[] as PoolListItem[])
 
-  const rewardAmounts: HumanTokenAmountWithAddress[] = useMemo(
-    () =>
-      allClaimableRewards.map(reward => ({
-        tokenAddress: reward.tokenAddress,
-        humanAmount: reward.humanBalance as HumanAmount,
-      })),
-    [allClaimableRewards]
-  )
+  const rewardAmounts: HumanTokenAmount[] = useMemo(() => {
+    return allClaimableRewards.map(reward => ({
+      tokenAddress: reward.tokenAddress,
+      humanAmount: reward.humanBalance as HumanAmount,
+      symbol: 'LP token',
```

Comment:
This is wrong. Reward amounts are not LP tokens. You need the actual symbol here.

Reply:
You are right. I had tested this and it was working as expected cause we don't really need the symbol here (I added it by mistake during the refactor) but we are getting it with `getToken` as we don't have the reward token symbol in the rewards onchain queries. 

Examples after removing: 

<img width="579" alt="Screenshot 2024-12-17 at 17 31 33" src="https://github.com/user-attachments/assets/dde5bafa-d172-4a4f-8625-241fa6bebe78" />

<img width="564" alt="Screenshot 2024-12-17 at 17 33 17" src="https://github.com/user-attachments/assets/d17bf726-3ca8-4480-baaa-592728da2e18" />